ansible/roles/gitlab_runner/defaults/main.yml

```diff
@@ -171,24 +171,38 @@ gitlab_runner__api_url: 'https://{{ gitlab_runner__api_fqdn }}/'
                                                                    # ]]]
 # .. envvar:: gitlab_runner__api_token [[[
 #
-# The personal GitLab API access token used for API operations, for example
-# removal of existing Runners. This is not a Runner registration token. You can
-# generate an access token in GitLab "User Settings", "Access Tokens"
+# The personal GitLab API access token used to register or remove the runners in GitLab.
+# You can generate an access token in GitLab "User Settings", "Access Tokens"
```

Comment:
Are the changes compatible with older GitLab versions, or just 17.x and higher?

Reply:
Compatible with GitLab >= 15.11.

> In GitLab 15.11 and later, you can use the [POST /user/runners REST API](https://docs.gitlab.com/ee/api/users.html#create-a-runner-linked-to-a-user) to create a runner as an authenticated user.

From: https://docs.gitlab.com/ee/ci/runners/new_creation_workflow.html#creating-runners-programmatically